packages/ui/src/components/modal/modal.tsx

```diff
@@ -86,7 +86,7 @@ export const Modal = ({
           {/* pr-8 at end on purpose so header never overflows icon */}
           {header && (
             <div
-              className={classes('relative', headerContainerClassName, 'pr-14')}
+              className={classes('relative', headerContainerClassName, 'pr-14', 'border-b border-b-secondary')}
```

Comment:
do we want to add the border-bottom to the modal?

Reply:
It's border bottom for this header
<img width="555" alt="Zrzut ekranu 2024-11-30 o 11 13 38" src="https://github.com/user-attachments/assets/f3758534-8df0-4910-8a25-10240da02012">

Reply:
oh, right. Thanks